scout/server/blueprints/cases/templates/cases/phenotype.html

```diff
@@ -62,21 +62,16 @@ <h6 class="mt-3 ms-3"><span class="fa fa-tag"></span>&nbsp;Cohort tag</h6>
     {% for diagnosis in case.diagnosis_phenotypes %}
       <div class="row">
         <div class="col-11">
-          {% if "OMIM" in diagnosis.disease_id and diagnosis.description %}
```

Comment:
OMIM should not be there any more and now there is a pydantic check that makes sure that all disease terms have a description

Reply:
Hm, but that is `diagnosis` on `case` - wasn't this for being compatible with an older revision?
But maybe we dont have any - we could check!

Reply:
Older versions had OMIM numbers saved, as integers. This is already taken care of in the controllers

Reply:
Ah, nice! Yes, looks like they should always be updated if ints. Ok, we just missed relaxing this back then I guess.